client/src/store/actions.js

```diff
@@ -0,0 +1,51 @@
+const Actions = {
+  SET_IS_ROOM_HOST: "SET_IS_ROOM_HOST",
+  SET_CONNECT_ONLY_WITH_AUDIO: "SET_CONNECT_ONLY_WITH_AUDIO",
+  SET_IDENTITY: "SET_IDENTITY",
+  SET_ROOM_ID: "SET_ROOM_ID",
+  SET_SHOW_OVERLAY: "SET_SHOW_OVERLAY",
+  SET_PARTICIPANTS: "SET_PARTICIPANTS",
+};

```

Comment:
You can extract the definitions of Action types into a separate file for better code organization.